tests/console/weechat.pm

```diff
@@ -27,6 +27,17 @@ sub run {
     select_console('user-console');
     script_run("weechat; echo weechat-status-\$? > /dev/$serialdev", 0);
     assert_screen('weechat');
+
+    type_string("/server add znc localhost/12345 -ssl -ssl_verify=0 -username=bernhard/freenode -password=$testapi::password\n");
+    assert_screen('weechat-server-added');
+
+    type_string("/connect znc\n");
+    assert_screen('weechat-no-longer-away');
+
+    type_string("/query *status\n");
+    type_string("Version\n");
```

Comment:
That could be a bit tricky as you want that command to show up in the query. Is it ensured to be open when you type this

Reply:
In my tests it never missed the query.
IIRC the query command doesn't require any server interaction so it should be quite fast.